Effects/Parallax/source/Flake.hx

```diff
@@ -0,0 +1,74 @@
+package;
+
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxMath;
+import flixel.math.FlxRandom;
+import flixel.util.FlxColor;
+
+class Flake extends FlxSprite
+{
+	private var _level:Int = 0;
+	private var _initialized:Bool = false;
+	
+	
+	public function new(Level:Int = 0) 
+	{
+		
+		super();
+		_level = Level;
+		revive();
+		_initialized = true;
+	}
+	
+	override public function revive():Void 
+	{
+		var size:Int = Std.int(FlxMath.bound(_level / 2, 1, 2));
+		makeGraphic(size, size, FlxColor.WHITE.getDarkened(.5 - (_level * .06)));
+		super.revive();
+		alpha = 1;
+		if (_initialized)
+		{
+			x = FlxG.random.int(0, FlxG.width * 4 * _level);
+			y = FlxG.random.int( -5, -10);
+		}
+		else
```

Comment:
It defaults to `false`, and `revive()` is called before it's set to `true` in the constructor.